src/main/java/org/takes/rq/RqMultipart.java

```diff
@@ -64,7 +64,14 @@
  */
 @SuppressWarnings("PMD.TooManyMethods")
 public interface RqMultipart extends Request {
-
+    /**
+     * Carriage return constant.
+     */
+    String CRLF = "\r\n";
```

Comment:
@2686747 this is not acceptable, see http://www.yegor256.com/2015/07/06/public-static-literals.html